io/read_delim.ts

```diff
@@ -19,7 +19,7 @@ function createLPS(pat: Uint8Array): Uint8Array {
       lps[i] = 0;
       i++;
     } else {
-      prefixEnd = lps[prefixEnd - 1];
+      prefixEnd = lps[prefixEnd - 1]!;
```

Comment:
`prefixEnd` is incremented inside the loop and `lsp` size is `pat.length` so this is non-null